app/components/unregistered-contributor-form.js

```diff
@@ -1,4 +1,6 @@
-import Ember from 'ember';
+import { get } from '@ember/object';
+import { computed } from '@ember/object';
```

Comment:
combine with line above

Reply:
Those are all caught by ESLint and will go into the same release. There is another PR branched off this one that is partially done.